Polish/spanie/spaniee.md

```diff
@@ -1,5 +1,5 @@
 Po obudzeniu się sytuacja się  nie zmieniła!
 
 Co robisz?
-
-[Start](../ptasieMleczko.md)
\ No newline at end of file
+[Całkowicie zmieniasz sytuację](zmieniasz/zmieniasz.md)
+[Start](../ptasieMleczko.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.